projects/plugins/boost/app/assets/src/js/global.d.ts

```diff
@@ -16,7 +17,7 @@ declare global {
 	};
 
 	// Constants provided by the plugin.
-	const Jetpack_Boost: {
+	const Jetpack_Boost: Refactoring_Critical_CSS_Recommendations & {
```

Comment:
`Refactoring_Critical_CSS_Recommendations` is this still meant to be called this?

Reply:
Good thing I called it `Refactoring` to have someone notice in code review 😊